styleguide/source/_patterns/05-pages/news.twig

```diff
@@ -16,7 +16,7 @@
 {% endblock %}
 
 {% block sidebarSecondary %}
-  {% include '@organisms/tools-block.twig' %}
-  {% include '@molecules/membership.twig' %}
+  {% include '@organisms/tools.twig' %}
+  {% include '@organisms/membership.twig' %}
```

Comment:
This should be `molecules/membership.twig` this was recently moved from an organism to a molecule.